README.md

```diff
@@ -27,8 +27,51 @@ Check out our [Contributing Guide] for how to get started.
     * [nothing yet...](#)
 
 # The List
-
-Nothing here yet...
+## Tooling ##
+
+### Register map over protocol code generator ###
+Often a MCU is connected to other devices (IC's) with protocols like I2C and SPI. These devices have their own register map. Register access for read and write is done together with providing the right address. Often these register maps are specified in datasheets (as PDF) and it's then left up to the programmar to implement everything in code. Sometimes C files are provided as well.
+
+This proposal is for a tool that can generate code for these register accesses. Ideally this should work for multiple protocols. Therefor it would be good to consider design for I2C and SPI (one of the two most common protocols) with each step of the implementation.
+
+### Implementation ###
```

Comment:
Could you please rewrite this section using the provided template? In general, I would prefer to specify "what you would like", rather than "how you think it should be done". This allows people to solve the problem in any way that makes sense, rather than just the way you originally imagined.

Reply:
There is plenty of room for people to implement it in any way that makes sense. This is just here to give people some guidance instead of throwing them into the deep.

Reply:
May i remind you that CONTRIBUTING guide speaks of "This lets someone know how to implement this request". I agree that guidance on implementation is useful. I'm not going to remove this section but i might rephrase it when i get specific suggestions.

Reply:
@flip111 in context, the guidance is:

> * Your request is clearly stated, linking to relevant documentation, such as a whitepaper, protocol definition, datasheet, etc.
>     * This lets someone know how to implement this request, and make Embedded Rust more awesome.

The idea is that your request is clearly stated (think "requirements"), and **that** is what helps people to implement this.

Reply:
I can rename "steps" to "requirements of parts" and leave out the hints of community discussion. But i don't see this as an improvement. Do you have a suggestion about rewrite?